AddressParser/AddressParser.cs

```diff
@@ -476,6 +482,9 @@ public string StatePattern
         /// </value>
         public string StreetPattern => @"
                         (?:
+                          # special case for addresses like 100 COUNTY ROAD F  http://regexstorm.net/tester?p=%28%3f%3a%28%3f%3cSTREET%3eCOUNTY%5cW%2bROAD%5cW*%5cw%2b%29%29&i=COUNTY+ROAD+FG%0d%0a%0d%0a&o=ixs
+                          (?:(?<STREET>COUNTY\W*ROAD\W*\w+))
+                          |
```

Comment:
Nevermind, its too early.  I was reading the results backward.  Your regex actually fixes, not causes problem.